google/cloud/spanner/internal/metadata_spanner_stub_test.cc

```diff
@@ -60,15 +59,27 @@ class MetadataSpannerStubTest : public ::testing::Test {
     EXPECT_EQ(TransientError(), status);
   }
 
+  Status IsContextMDValid(grpc::ClientContext& context,
+                          std::string const& method) {
+    return validate_metadata_fixture_.IsContextMDValid(
+        context, method, google::cloud::internal::ApiClientHeader(),
+        db_.FullName());
+  }
+
+  std::multimap<std::string, std::string> GetMetadata(
+      grpc::ClientContext& context) {
+    return validate_metadata_fixture_.GetMetadata(context);
+  }
+
   template <typename MockCall, typename Request, typename MemberFunction>
   void ExpectSession(MockCall& call, Request const&,
                      std::string const& rpc_name,
                      MemberFunction member_function) {
     call.WillOnce(
         [this, rpc_name](grpc::ClientContext& context, Request const&) {
-          EXPECT_STATUS_OK(
-              IsContextMDValid(context, "google.spanner.v1.Spanner." + rpc_name,
-                               expected_api_client_header_, db_.FullName()));
+          EXPECT_STATUS_OK(validate_metadata_fixture_.IsContextMDValid(
```

Comment:
why don't we use the helper method here?

Reply:
Fixed.